src/index.js

```diff
@@ -1,13 +1,129 @@
+import {
+  BALL,
+  STRIKE,
+  NOTHING,
+  ERROR,
+  CORRECT,
+  RESTART,
+  RESTART_BTN,
+} from './define.js';
+
+const baseballGame = new BaseballGame();
+const computerInputNumbers = generateRandomNumbers();
+const userInputNumbers = document.getElementById('user-input');
```

Comment:
```suggestion
const $userInputNumbers = document.getElementById('user-input');
```
저는 DOM 관련 할당을 구분하기 위해 $를 추가했는데 좋은지 잘 모르겠어서 어떻게 생각하시는지 궁금해서 리뷰달았습니다.

Reply:
맞아요. 관련 개념이 전혀 없었는데 규칙을 정해서 구분하는게 더 좋을 것 같아요.